book/html.md

```diff
@@ -245,11 +237,15 @@ works!
 ::: {.further}
 The ill-considered Javascript `document.write` method allows
 Javascript to modify the HTML source code while it's being parsed!
-Modern browsers use [speculative][speculative-parsing] parsing to
-make this fast and avoid evaluating Javascript while parsing.
+This is actually a [bad idea][document-write-bad]. A naive
+implementation of `document.write` would have the HTML parser stop to
+execute JavaScript, but that would be very slow. In fact, modern
+browsers use [speculative][speculative-parsing] parsing to make this
+fast and avoid evaluating Javascript while parsing.
```

Comment:
"
```suggestion
fast.
```

This phrase is not accurate / meaningless..

Reply:
In general the parser does actually stop for all scripts. The speculative parser is just to find subresources to load faster.